test/agenda.js

```diff
@@ -416,7 +410,10 @@ describe("Agenda", () => {
             })
             .schedule("now")
             .save();
+            debug('Job 2 scheduled for ' + job2.attrs.nextRunAt.toISOString())
 
+          // TODO: This test is currently failing, and I have to ask whether the test describes the expected behavior
```

Comment:
Please remove this comment. It's not relevant any more. The unit test got broken with a different PR. The test is passing now. And the test is correct. It must make sure that the `nextRunAt` changed. Later on it checks that there is only one job in the DB.